tests/dataset/test_aoi.py

```diff
@@ -114,10 +116,7 @@ def test_stac_url_input(self):
         assert aoi.download_data is True
         assert Path("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif").is_file()
         aoi.close_raster()
-        try:
-            os.remove("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif")
-        except PermissionError:
-            pass
+        os.remove("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif")
```

Comment:
I think this kind of exceptions could be left unchanged because it really helps when testing locally under Windows OS